diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4cd75cf..d594915 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -239,7 +239,7 @@ include_directories(${CBLAS_INCLUDE_DIR})
 # ========================
 check_cxx_compiler_flag("-Werror -Wall -Wno-unused-result -Wno-sign-compare -Wno-literal-suffix" HAVE_GCC_WARNING_FLAGS)
 if(HAVE_GCC_WARNING_FLAGS)
-  SET(CMAKE_CXX_FLAGS "-Werror -Wall -Wno-unused-result -Wno-sign-compare -Wno-literal-suffix ${CMAKE_CXX_FLAGS}")
+  SET(CMAKE_CXX_FLAGS "-Wall -Wno-unused-result -Wno-sign-compare -Wno-literal-suffix ${CMAKE_CXX_FLAGS}")
   SET(CMAKE_C_FLAGS "-Werror -Wall -Wno-sign-compare ${CMAKE_C_FLAGS}")
 endif()
 check_cxx_compiler_flag("-Wno-undefined-var-template" DISABLE_CLANG_FALSE_POSITIVES)
